background/constants/index.ts

```diff
@@ -23,9 +23,12 @@ export const MINUTE = 60 * SECOND
 export const HOUR = 60 * MINUTE
 export const DAY = 24 * HOUR
 
-export const COMMUNITY_MULTISIG_ADDRESS =
+export const DEFAULT_COMMUNITY_MULTISIG_ADDRESS =
   "0x99b36fDbC582D113aF36A21EBa06BFEAb7b9bE12"
 
+// TODO add correct addresses
+export const COMMUNITY_MULTISIG_ADDRESS: Record<string, string> = {}
```

Comment:
I think we can be more explicit in what this contains by expanding the type and adjusting the name a bit:

```suggestion
export const COMMUNITY_MULTISIG_ADDRESS_BY_CHAINID: {
  [chainID: string]: string
} = {}
```